src/parser/jobs/brd/modules/RefulgentProcs.tsx

```diff
@@ -1,9 +1,17 @@
+import {t} from '@lingui/macro'
 import {Trans} from '@lingui/react'
 import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {dependency} from 'parser/core/Injectable'
 import {Procs} from 'parser/core/modules/Procs'
-import React from 'react'
+import {Timeline} from 'parser/core/modules/Timeline'
+import React, {Fragment} from 'react'
+import {Table, Button} from 'semantic-ui-react'
 
 export class RefulgentProcs extends Procs {
+	@dependency private timelinemodule!: Timeline
```

Comment:
This dependency already exists in the parent Procs module, could you set it to `protected` there instead of re-declaring it?